test/distributed/test_c10d.py

```diff
@@ -29,7 +29,7 @@
 from torch.testing._internal.common_distributed import MultiProcessTestCase, \
     requires_gloo, requires_nccl, requires_nccl_version, \
     skip_if_not_multigpu, skip_if_lt_x_gpu, get_timeout, skip_if_rocm, \
-    skip_if_rocm_single_process, simple_sparse_reduce_tests, skip_if_win32, \
+    simple_sparse_reduce_tests, skip_if_win32, \
```

Comment:
Are these changes intended as part of this PR? If so, could you update the PR summary to mention why we are removing these skip decorators?